### PR TITLE
Bz1600199

### PR DESCRIPTION
**Pull request by prashant-redhat**
@openshift/team-documentation: Bz1600199: Corrected the role clusteradmin to cluster-admin and rephrased sentences as per style guide. Peer review needed.

### CODE REVIEW

File: admin_guide/manage_rbac.adoc [hunk 1]
```diff
@@ -278,7 +278,7 @@ PolicyRule:
 ----
 endif::[]
 
-To view the current set of cluster role bindings, which show the users and
+	To view the current set of cluster role bindings, which show the users and
```
Comment (kalexand-rh) [inline admin_guide/manage_rbac.adoc]: Double-check this white space.

File: admin_guide/manage_rbac.adoc [hunk 2]
```diff
@@ -768,18 +768,17 @@ $ oc adm policy add-role-to-user podview user2 --role-namespace=bob-project -n b
 A cluster role binding is a binding that exists at the cluster level.
 A role binding exists at the project level. The cluster role _view_ must be
 bound to a user using a local role binding for that user to view the project.
-Local roles should only created if a cluster role does not provide the set
+Local roles should only be created if a cluster role does not provide the set
```
Comment (kalexand-rh) [inline admin_guide/manage_rbac.adoc]: s/Local roles should only be created/Create local roles only

File: admin_guide/manage_rbac.adoc [hunk 3]
```diff
@@ -768,18 +768,17 @@ $ oc adm policy add-role-to-user podview user2 --role-namespace=bob-project -n b
...
+`cluster-admin` to a user using a local role binding, making it appear
 that this user has the privileges of a cluster administrator. This is not the case.
-The `cluster-admin` cluster role bound to a certain project is more like a super
-administrator for that project, granting the permissions of the cluster role
+Binding the `cluster-admin` cluster role to a certain project grants the permissions of the cluster role
 `admin`, plus a few additional permissions like the ability to edit rate limits.
 This can appear especially confusing via the web console UI, which does not list
-cluster role bindings (which are bound to true cluster administrators). However, it
-does list local role bindings (which could be used to locally bind `cluster-admin`).
+cluster role bindings which are bound to true cluster administrators. However, it
```
Comment (kalexand-rh) [inline admin_guide/manage_rbac.adoc]: s/which/that

**Review by kalexand-rh — COMMENTED** [review]
@prashant-redhat, it looks like your PR picked up more than you intended. Will you PTAL?

**Comment by vikram-redhat** [comment]
@prasanth-RH - let me know if you are having issues updating it. If it will make it easier, let's just close this PR and then open a new one.

**Comment by prashant-redhat** [comment]
@vikram-redhat this one is a new PR. I had closed the previous one as suggested by @kalexand-rh . I will try to update this PR else create a new one.

**Comment by vikram-redhat** [comment]
@prasanth-RH - ah looks like this hasn't worked correctly. I am going to close this PR. Please reopen a new PR as per the manual.

**Pull request closed without merging**